migration_tools.drush.inc

```diff
@@ -248,6 +263,51 @@ function drush_migration_tools_mt_migrate_file_folders($organization) {
 }
 
 /**
+ * Drush command callback for migration-tools-html-file-list.
+ *
+ */
+function drush_migration_tools_html_file_list($directory) {
+  // Set up options
+  $format = drush_get_option('format');
+  $outfile = drush_get_option('outfile');
+  $mask = '/\.htm/';
```

Comment:
I think this is the mask you want here so it is case insensitive
`'/.*\.htm(l)?$/i'`

and maybe  
```
$mask = drush_get_option('outfile');
$mask = (empty($mask)) ? '/.*\.htm(l)?$/i' : $mask;
```